sdk/storage/azure-storage-blob/azure/storage/blob/_blob_service_client.py

```diff
@@ -416,6 +416,36 @@ def list_containers(
                 page_iterator_class=ContainerPropertiesPaged
             )
 
+    @distributed_trace
+    def filter_blobs(self, where=None, **kwargs):
+        # type: (Optional[str], Optional[Any], **Any) -> ItemPaged[BlobProperties]
+        """The Filter Blobs operation enables callers to list blobs across all
+        containers whose tags match a given search expression.  Filter blobs
+        searches across all containers within a storage account but can be
+        scoped within the expression to a single container.
+
+        :param str where:
+            Filters the results to return only to return only blobs
+            whose tags match the specified expression.
+        :keyword int results_per_page:
+            The max result per page when paginating.
+        :keyword int timeout:
+            The timeout parameter is expressed in seconds.
+        :returns: An iterable (auto-paging) response of BlobProperties.
+        :rtype: ~azure.core.paging.ItemPaged[~azure.storage.blob.BlobProperties]
```

Comment:
Should this be `~azure.core.paging.ItemPaged[~azure.storage.blob.FilteredBlob]`?